vagrant/hosts

```diff
@@ -4,7 +4,7 @@
 
 [all:vars]
 ansible_ssh_user=vagrant
-ansible_dns=mesos.dev
+ansible_domainname=mesos.dev
```

Comment:
Yes, I think this is a better name. And maybe we should even avoid the `ansible_` namespace and use `mesos_domain`.